crates/swift-bridge-ir/src/bridged_type.rs

```diff
@@ -1158,9 +1158,25 @@ impl BridgedType {
                         unimplemented!()
                     }
                 },
-                StdLibType::Vec(ty) => {
-                    format!("RustVec<{}>", ty.ty.to_swift_type(type_pos, types))
-                }
+                StdLibType::Vec(ty) => match type_pos {
+                    TypePosition::FnArg(func_host_lang, _) => {
+                        if func_host_lang.is_rust() {
+                            format!("RustVec<{}>", ty.ty.to_swift_type(type_pos, types))
+                        } else {
+                            "UnsafeMutableRawPointer".to_string()
+                        }
+                    },
```

Comment:
Need to run `cargo fmt` https://github.com/chinedufn/swift-bridge/actions/runs/5272573390/jobs/9561689835